nautobot/extras/templates/extras/customfield.html

```diff
@@ -101,6 +101,14 @@
                             </ul>
                         </td>
                     </tr>
+                    <tr>
+                        <td>Grouping</td>
```

Comment:
I think it'd make more sense to have this located right after `Label` in the display.